rel-eng/openshift-master.service

```diff
@@ -2,6 +2,11 @@
 Description=OpenShift Master
 Documentation=https://github.com/openshift/origin
 After=network.target
+After=etcd.service
+Wants=etcd.service
```

Comment:
This does mean that if etcd were installed, but we're configured to use the embedded etcd, it would attempt to start etcd and we'd run into problems binding to the port assuming it's not reconfigured to use another port. But, uhhh, don't install etcd if you're not going to use it?